assets/queries/cloudFormation/memcached_disabled/test/positive1.yaml

```diff
@@ -0,0 +1,13 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+  ElasticacheCluster:
```

Comment:
Change "ElasticacheCluster" to "ElasticacheCluster3"

Reply:
done